sdk/api-reference.md

```diff
@@ -6,152 +6,95 @@ icon: terminal
 
 ## Builder API Reference
 
-This document specifies the fluent builder APIs used to create Doppler auctions. Builders assemble type‑safe parameter objects for `DopplerFactory.createStaticAuction` and `DopplerFactory.createDynamicAuction`, applying sensible defaults and computing derived values (ticks, gamma) where helpful.
+Builders assemble type‑safe parameter objects for `DopplerFactory.createStaticAuction`, `DopplerFactory.createDynamicAuction`, and `DopplerFactory.createMulticurve`.
 
 * Static auctions: Uniswap V3 style, fixed price range liquidity bootstrapping
 * Dynamic auctions: Uniswap V4 hook, dynamic Dutch auction with epoch steps
-
-All types referenced are exported from `src/types.ts`.
+* Multicurve auctions: Uniswap V4 initializer with multiple curves
 
 ### Common Concepts
 
-* Token specification:
-  * `standard` (default): DERC20 with optional vesting and yearly mint rate
-* Governance is required:
-  * Call `withGovernance(...)` in all cases.
-  * `withGovernance()` with no arguments applies standard governance defaults.
-  * `withGovernance({ noOp: true })` explicitly selects no‑op governance (requires chain support).
-  * Or provide `initialVotingDelay`, `initialVotingPeriod`, and `initialProposalThreshold`, or `withGovernance({ useDefaults: true })`.
+* Governance is required - call `withGovernance(...)` in all cases
```

Comment:
This doesn't make sense to me. In general, governance is optional. If a user doesn't supply .withGovernance() params it should default to false and not have governance.